tools/rpkg/inst/include/cpp11/as.hpp

```diff
@@ -2,15 +2,15 @@
 // vendored on: 2022-01-10
 #pragma once
 
-#include <cmath>             // for modf
-#include <initializer_list>  // for initializer_list
-#include <memory>            // for std::shared_ptr, std::weak_ptr, std::unique_ptr
+#include <cmath>            // for modf
```

Comment:
I think this is also third party code - probably want to avoid editing it

Reply:
I think this is heavily used by the R pkg? But I'll undo the change and see if anything breaks